pkgs/top-level/all-packages.nix

```
@@ -34881,7 +34881,13 @@ with pkgs;
 
   pyrosimple = callPackage ../applications/networking/p2p/pyrosimple { };
 
-  qbittorrent = libsForQt5.callPackage ../applications/networking/p2p/qbittorrent { };
+  qbittorrent = qbittorrent-qt5;
+  qbittorrent-qt5 = libsForQt5.callPackage ../applications/networking/p2p/qbittorrent {
```

```suggestion
  qbittorrent = libsForQt5.callPackage ../applications/networking/p2p/qbittorrent {
```

Add remove `qbittorrent = qbittorrent-qt5;`, do not add unnecessary aliases

---

I disagree that it's an unnecessary alias. It will allow people using the Qt5 variant to seamlessly transition to the new version. The Qt6 variant is opt-in because of the theming issues mentioned above.